src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/utils/NatureOfControlValidators.java

```diff
@@ -12,9 +13,11 @@ public class NatureOfControlValidators {
 
     private NatureOfControlValidators() {}
 
-    public static boolean checkAtLeastOneFieldHasValue(List<NatureOfControlType> fields, String qualifiedFieldName, Errors errs, String loggingContext) {
+    //UAR-1595 remove last feature flag parameter when FEATURE_FLAG_ENABLE_PROPERTY_OR_LAND_OWNER_NOC_30082024 is removed
+    public static boolean checkAtLeastOneFieldHasValue(List<NatureOfControlType> fields, List<NatureOfControlJurisdictionType> jurisdictionFields, String qualifiedFieldName, Errors errs, String loggingContext, boolean isPropertyAndLAndNocEnabled) {
```

Comment:
It would be preferable to keep the parameters relating to business logic first and keep errors and logginContext in last place

Reply:
Rearranged parameters and arguments.